src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +39,7 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" onClick={this.handleBackButtonClick} to="/">Retour</Link>
```

Comment:
Issue: if I arrive on the details page from a direct link, this “go back” button doesn't mean anything to me.
Solution: “See the ${research results count - 1} other variables and parameters”.